src/renderer/components/depositActions/forms/Forms.styles.tsx

```diff
@@ -0,0 +1,56 @@
+import * as A from 'antd'
+import styled from 'styled-components'
+import { palette } from 'styled-theme'
+
+import { media } from '../../../helpers/styleHelper'
+import { Button as UIButton } from '../../uielements/button'
+import { Label as UILabel } from '../../uielements/label'
+
+export const Form = styled(A.Form)`
```

Comment:
Use `InnerForm` imported from '../../shared/form/Form.style' to avoid type casting issues with `onFinish` you have in components.